pcdsdevices/epics_motor.py

```diff
@@ -152,6 +154,50 @@ def disable(self):
         """
         return self.disabled.set(value=1)
 
+    def stop(self):
+        """
+        Stops the motor.
+
+        Returns
+        -------
+        status: Status Object
+            Status object of the set
+        """
+        return self.motor_spg.put(value='Stop')
+
+    def pause(self):
+        """
+        Pauses a move.
+
+        Returns
+        -------
+        status: Status Object
+            Status object of the set
+        """
+        return self.motor_spg.put(value='Pause')
+
+    def resume(self):
+        """
+        Sets motor ready to move or resumes a paused move (same as <motor>.go())
+
+        Returns
+        -------
+        status: Status Object
+            Status object of the set
+        """
+        return self.motor_spg.put(value='Go')
+
+    def go(self):
+        """
+        Sets motor ready to move or resumes a paused move.
+
+        Returns
+        -------
+        status: Status Object
+            Status object of the set
+        """
+        return self.motor_spg.put(value='Go')
```

Comment:
I am uncomfortable with a `method` called `go` that doesn't neccesarily `move` something. I'd leave it at just `resume`. Just a thought.

Reply:
```go```  was added in because it matches what staff are used to on the motor expert screen.  It really does make a motor *go* provided a move has been set and paused.  Maybe we can keep it around and see if people don't like it?

About setting the motor SPG during Bluesky staging, I'm not sure if we want to do this if the motor is stopped.  Staff often stop motors as a way of "soft-disabling" them because they haven't been completely set up yet or they don't want them moved for whatever reason.  I think its better to display a transparent error message about a motor being stopped and let them set it themselves than to assume its OK to move and go ahead.

Reply:
However I don't see anything wrong with calling ```go``` during a stage if the motor is paused.